### PR TITLE
Updating Resource Links on Getting Started pages

### PR DESCRIPTION
**Pull request by lparrott**
- [W-11431598](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE000011TxLaYAK/view)
- Quip doc with changes: https://salesforce.quip.com/ifaDA9SElkby

# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# Features Intended for Future Release

# Features for Elevate Customers

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 1]
```diff
@@ -16031,7 +16031,7 @@ Note: The NPSP Settings tab is visible in the Nonprofit Success Pack application
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>Questions Answered link label in Get Started Resources section for End Users</shortDescription>
-        <value>Trailblazer Community</value>
```
Comment (andrewyu-salesforce) [inline force-app/main/default/labels/CustomLabels.labels-meta.xml:16034]: Is there a reason this specific label is switching back to Trailblazer Community? I see the link corresponding to this label is still redirected to the Trailblazer Community page.

**Review by andrewyu-salesforce — COMMENTED** [review]
@lparrott Just one question on the label change

File: force-app/main/default/labels/CustomLabels.labels-meta.xml (hunk 1 above)
Comment (lparrott) [reply force-app/main/default/labels/CustomLabels.labels-meta.xml:16034]: Great question, I took this from a Quip Doc that was provided, I just asked for confirmation of the change there.

Comment (daniel-fuller) [reply force-app/main/default/labels/CustomLabels.labels-meta.xml:16034]: yeah it was my understanding that we've now changed all URLs to point to Trailblazer Community and not Power of Us Hub

Comment (lparrott) [reply force-app/main/default/labels/CustomLabels.labels-meta.xml:16042]: I haven't heard back about this, but I also think we should keep it as Trailblazer Community, this was probably a mistake in the Doc that I was working from. I'll make that revert tomorrow!

**Review by daniel-fuller — APPROVED** [review]

**Pull request merged**